chemprop/v2/tests/unit/featurizers/test_molgraph.py

```diff
@@ -72,10 +72,10 @@ def test_atom_fdim(extra):
 
 
 def test_bond_fdim(atom_messages, extra):
-    mf = MoleculeMolGraphFeaturizer(extra_bond_fdim=extra, bond_messages=atom_messages)
+    mf = MoleculeMolGraphFeaturizer(extra_bond_fdim=extra, bond_messages= not atom_messages)
 
     if atom_messages:
-        assert mf.bond_fdim == len(mf.bond_featurizer) + extra
+        assert mf.bond_fdim == len(mf.bond_featurizer) + extra 
```

Comment:
Could you remove the extra blank

Reply:
`bond_messages= not atom_messages` to `bond_messages=not atom_messages` as well

Reply:
I have made both of these changes.